### PR TITLE
Adjust the implementation of getDynamicConfigurationDouble and DynamicConfigurationBoolean

### PR DESCRIPTION
**Pull request by Technoboy-**
### Motivation
1. Adjust the implementation of getDynamicConfigurationDouble and DynamicConfigurationBoolean.
2. Fix typos.

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java [hunk 1]
```diff
@@ -317,29 +316,11 @@ private String getDynamicConfigurationFromStore(String path, String settingName,
...
-        try {
-            String setting = this.getDynamicConfigurationFromStore(path, settingName, null);
-            if (setting != null) {
-                result = Double.parseDouble(setting);
-            }
-        } catch (Exception e) {
-            log.warn("Got exception when parsing configuration from path [{}]:", path, e);
-        }
-        return result;
+        return Double.parseDouble(getDynamicConfigurationFromStore(path, settingName, String.valueOf(defaultValue)));
```
Comment (eolivelli) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java:319]: with this new code if there is something wrong written on the store we will throw a NumberFormatException
we should keep the "catch" statement

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/UnAckedMessageTracker.java [hunk 2]
```diff
@@ -150,7 +150,7 @@ public void run(Timeout t) throws Exception {
         }, this.tickDurationInMs, TimeUnit.MILLISECONDS);
     }
 
-    public static void addChunkedMessageIdsAndRemoveFromSequnceMap(MessageId messageId, Set<MessageId> messageIds,
+    public static void addChunkedMessageIdsAndRemoveFromSequenceMap(MessageId messageId, Set<MessageId> messageIds,
```
Comment (eolivelli) [inline pulsar-client/src/main/java/org/apache/pulsar/client/impl/UnAckedMessageTracker.java:153]: I am not sure it is a good practice to change this unrelated code with this patch.
this patch is focused on LoadBalancer.
it is better to still to change things only in one module

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: pulsar-client/src/main/java/org/apache/pulsar/client/impl/UnAckedMessageTracker.java (hunk 2 above)
Comment (Technoboy-) [reply pulsar-client/src/main/java/org/apache/pulsar/client/impl/UnAckedMessageTracker.java:153]: Ok, I will update later.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java (hunk 1 above)
Comment (Technoboy-) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java:319]: yes，I will rollback.

**Review by eolivelli — APPROVED** [review]
LGTM

**Review by sijie — APPROVED** [review]
